### ISSUE TITLE
The VoiceOver is announcing as Image for the Tab elements (i.e., Console/iOS/Android/Mac) of the Create a new workbooks.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#267]
### Steps to Reproduce

1. Launch the Xamarin Workbooks, and Turn ON the Voiceover AT.
2. "Create a new Workbook" window-> Navigate to the Console/iOS/Android/Mac tab which is just above the "Create Workbook" button.

### Expected Behaviour
The Voiceover should announce it as "Console/iOS/Android/Mac" Tab.

### Actual Behaviour
The Voiceover is announcing it as "image".
The Screen-reader user won't get purpose of the item until its control type  is not defined correctly.

### User Impact:
The Screen-reader user won't get purpose of the item until its control type  is not defined correctly.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)We saved this conversation. You'll see it soon in the Conversations tab in Skype for Business and in the Conversation History folder in Outlook. 

### Log File

[VO1_Tabs_Image.zip](https://github.com/Microsoft/workbooks/files/1990832/VO1_Tabs_Image.zip)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1822
